Blackjack.Tests/ControllerShould.cs

```diff
@@ -40,9 +40,16 @@ public void ReturnDealerWin_GivenPlayerBust()
                 .Returns(new Card(CardRank.Two, CardSuit.Clubs))
                 .Returns(new Card(CardRank.Four, CardSuit.Diamonds));
 
-            controller.Play();
+            var playerHand = player.Hand;
+            var dealerHand = dealer.Hand;
+            var expectedGameResult = new GameResult(dealerHand, playerHand);

```

Comment:
here we want to 'hard code' a dealer hand and player hand as these are the expected hands. Calling player.Hand and dealer.Hand produces the actual hands (the hands that the player and dealer ends up with as a result of calling controller.play).

Reply:
We're essentially hard coding the hands by mocking the return values for the `deck.DealCard()` calls @indianbread